tests/fast-integration/eventing-test/image/Dockerfile

```diff
@@ -0,0 +1,26 @@
+FROM debian:11
+
+ENV KUBECONFIG=/KUBECONFIG/kubeconfig.yaml
+ENV EVENTMESH_SECRET_FILE=/EMS/ems.json
```

Comment:
Since now the make target can be anything provided to the container, hardcoding more env vars doesn't have much of a value I think. The env variables could anyway be provided in the Job. So I think for now this would be enough.

Reply:
Alright, let's wrap it up then :)